packages/@storefront/flux-capacitor/CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [major|minor|patch]
+### Added
+- SF-1355:
```

Comment:
Distribute this into the two sub-points and promote the sub-points. The main point isn't the ticket, it's the things done.

Reply:
Good point, will do.